x-pack/plugins/security/server/authentication/providers/pki.ts

```diff
@@ -272,9 +272,13 @@ export class PKIAuthenticationProvider extends BaseAuthenticationProvider {
 
     let result: { access_token: string; authentication: AuthenticationInfo };
     try {
-      result = await this.options.client.callAsInternalUser('shield.delegatePKI', {
-        body: { x509_certificate_chain: certificateChain },
-      });
+      result = (
+        await this.options.client.asInternalUser.transport.request({
```

Comment:
__note:__ hopefully we can publish spec for this API (discussed in https://github.com/elastic/elasticsearch/issues/67189)